src/components/Main/MovieCardMenu.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import Modal from '../Modal';
+import EditMovieForm from './EditMovieForm';
+import DeleteMovieForm from './DeleteMovieForm';
+
+class MovieCardMenu extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            isMenuActive: false,
+            form: null,
+            modalType: null
+        };
+    }
+
+    showModal = (form) => {
+        this.setState({
+            showModal: true,
+            form
+        });
+    };
+
+    onModalClose = () => {
+        this.setState({
+            modalType: null,
+            showModal: false
+        });
+    };
+
+    render() {
+        return (
+            <div className={this.state.isMenuActive ? 'movie-card__menu active' : 'movie-card__menu'}>
+                <button className="movie-card__menu-open-btn" onClick={() => this.setState({isMenuActive: true})}/>
+                <div className={this.state.isMenuActive ? 'movie-card__menu-popup active' : 'movie-card__menu-popup'}>
+                    <ul className="movie-card__menu-popup-list">
+                        <li className="movie-card__menu-popup-list-item"
+                            onClick={() => this.showModal(<EditMovieForm movie={this.props.movie}/>)}
```

Comment:
we should never store any jsx in state! 😨  it's against React nature

Reply:
try to store for example `modalType` (which can be equal to 'edit', 'delete', 'add')
and then
```
{this.state.showModal &&
    <Modal onModalClose={this.onModalClose}>
        {this.state.modelType === 'edit' && <EditMovieForm movie={this.props.movie}/>}
        {this.state.modelType === 'delete' && <EditMovieForm movie={this.props.movie}/>}
    </Modal>
}
```
but please please please, DO NOT save jsx into state

Reply:
Got it, thank you very much for helping me! Fixed.